prac_06/guitar.py

```diff
@@ -1,8 +1,30 @@
+"""CP1404 Practical 6- Guitars!
+Tahlia Clennar
+Estimated time: 25 minutes
+Actual time: 15 minutes
+"""
+
+YEAR = 2022
+VINTAGE_YEAR = 50
+
+
 class Guitar:
     """Represent programming language."""
```

Comment:
Does this represent programming language?

Reply:
Whoops thanks for pointing that out. I forgot to change my docstring when copying some lines from the other file.